### DISCUSSION TITLE
Highlight cucumber steps description

### DISCUSSION THREAD

**Discussion by amilcarjustino** [badeball/cypress-cucumber-preprocessor#915] [General]
Hello guys.

I would like to ask, if it is somewhat possible to change the bold or color of the cucumber steps description while running the tests. 
Using the image as reference, i would like to highlight the steps that I marked red, so that, when debugging, I can see more clearly which step is failing.

<img width="445" alt="Screenshot 2023-01-04 at 15 54 03" src="https://user-images.githubusercontent.com/92743525/210597142-282ec1b0-3ab9-4983-a45a-726220f0ef21.png">

Kind Regards
Amílcar

**Comment by badeball** [comment]
Duplicate of https://github.com/badeball/cypress-cucumber-preprocessor/issues/796
